tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -632,7 +629,14 @@ module.exports = {
       locateStrategy: 'xpath'
     },
     noResharePermissions: {
-      selector: '#oc-files-sharing-sidebar .files-collaborators-no-reshare-permissions-message'
+      selector:
+        '//div[@id="oc-files-sharing-sidebar"]//p[@data-test-id="files-collaborators-no-reshare-permissions-message"]',
```

Comment:
The idea of the `data-test-id` is that it's ideally unique across the page. So IMO sufficient to have the following selector:
```suggestion
        '//*[@data-test-id="files-collaborators-no-reshare-permissions-message"]',
```